units/src/amount/unsigned.rs

```diff
@@ -50,10 +52,18 @@ mod encapsulate {
     pub struct Amount(u64);
 
     impl Amount {
+        /// The maximum value of an amount.
+        pub const MAX: Self = Self(21_000_000 * 100_000_000);
+        /// The minimum value of an amount.
+        pub const MIN: Self = Self(0);
+
         /// Constructs a new [`Amount`] with satoshi precision and the given number of satoshis.
         ///
-        /// Caller to guarantee that `satoshi` is within valid range. See [`Self::MAX`].
-        pub const fn from_sat_unchecked(satoshi: u64) -> Amount { Self(satoshi) }
+        /// Accepts an `u32` which is guaranteed to be in range for the type, but which can only
```

Comment:
```suggestion
        /// Accepts a `u32` which is guaranteed to be in range for the type, but which can only
```

Reply:
Can you please post what commit you're commenting on? Github shows 3 or 4 lines of context and nothing else so it is very hard to figure out what change you're talking about without further hints.

Reply:
It's 61c54aa4fb73d38d0cd55af4148cb54155e424f7

But it's unrelated, so not needed for this PR. Could be added to the typos list.

Reply:
It's just a minor nit on grammer: https://github.com/rust-bitcoin/rust-bitcoin/pull/4256/commits/beaa2db7e523ca1ac2bb6c5eaf22f8c42db67f5c#diff-0ced1794ac364cd494a4351ae6baed7b4d14d3dc673b945f3eae8f83e4e93844R62

Should be `a` instead of `an` since followed by something that is or sounds like a vowel.

Reply:
nm this is now `i32`

Reply:
should still be `a` I think but feel free to disregard my nit picking of course.